docs/BEMAN_LIBRARY_MATURITY_MODEL.md

```diff
@@ -11,30 +11,24 @@ SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 ### Under development and not yet ready for production use.
 <img src="../images/logos/beman_logo-beman_library_under_development.png" style="width:5%; height:auto;"> These libraries may deviate from the Beman Standard due to incompleteness, lack of testing, inconsistencies with the specification, or other non-conformances.
 
-> [!CAUTION]
```

Comment:
This syntax is not supported by the website. As it's not so important and we don't have a solution, I propose to remove this extra annotations.